CONTRIBUTING.md

```diff
@@ -33,3 +33,24 @@ npm install
 npm test
 ```
 
+# Coding Guidelines
+
+## License header
+
+All source files must begin with a comment of the below header:
+
+```go
+// Copyright 2017 Google Inc.
```

Comment:
Could replace 2017 with {YEAR} so that we don't have to update this later.

Also this doesn't make it clear which files are source files. I'd guess .js and .go get the header, whereas .html and .css files don't?